week-14/practices/12-context-practices/practice-1-README.md

```diff
@@ -1,56 +1,63 @@
 # Practice: Context Provider with Dynamic Value Practice
 
-In this practice, you will:
-
-* Set up a context provider to use state to hold a dynamic value for the context
-
-## Phase 0: Setup
-
-Download the [starter][context-starter] file for this practice.
-
-Open the starter in VSCode and run `npm install` in the terminal.
-
-Run `npm start` to make sure your project is set up correctly.
+In our previous practice, we have created our context and consumed it in our
+`Detail` component. Let's now make this a more dynamic and show you how context
```

Comment:
**Line 4:** "...a more dynamic and..." 

Don't leave me on a cliffhanger

Reply:
What do you think about this? "Let's now make our value more dynamic and show you how 
context can be used!"